libr/core/panels.c

```diff
@@ -2590,7 +2590,12 @@ void __move_panel_to_down(RCore *core, RPanel *panel, int src) {
 	for (; i < panels->n_panels - 1; i++) {
 		RPanel *tmp = __get_panel (panels, i);
 		int t_y = ((double)tmp->view->pos.y / (double)h) * (double)new_h + 1;
-		int t_h = ((double)tmp->view->pos.h / (double)h) * (double)new_h + 1;
+		int t_h;
+		if (tmp->view->edge & (1 << PANEL_EDGE_BOTTOM)) {
```

Comment:
us ternary operator and make it const, same for t_y.

also i think it should be size_t because no x/y coordinates can be negative here